workflows/promote-job.ts

```diff
@@ -0,0 +1,103 @@
+import fs from 'fs-extra';
+import {cyan, green, red} from 'kleur';
```

Comment:
Colors don't show up and it would make this file simpler if we took it out of the logging.

Reply:
Done b1efb9d